README.rst

```diff
@@ -2,6 +2,16 @@
 conda-build
 ===========
 
+Installation
+--------------
+.. code:: bash
+
+    # Display information about current conda install
+    conda info
+
+    # Install conda-build in the current env
```

Comment:
conda-build can only be installed into the root environment. Perhaps the install command should be `conda install -n root conda-build`.